File: lib/models/article.dart
class Article {
  final int id;
  final String tag;
  final String title;
  final String readTime;
  final String content;

  Article({
    required this.id,
    required this.tag,
    required this.title,
    required this.readTime,
    required this.content,
  });

  factory Article.fromJson(Map<String, dynamic> json) {
    return Article(
      id: json['id'],
      tag: json['tag'] ?? '',
      title: json['title'] ?? '',
      readTime: json['read_time'] ?? '',
      content: json['content'] ?? '',
    );
  }
}

File: app/Models/Article.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;

class Article extends Model
{
    protected $fillable = [
        'tag',
        'title',
        'content',
        'read_time',
    ];

    public function notifications()
    {
        return $this->hasMany(UserNotification::class);
    }
}
